axel_lab_to_nwb/conversion_module.py

```diff
@@ -60,120 +81,143 @@ def conversion_function(source_paths, f_nwb, metadata, plot_rois=False, **kwargs
+            )
+            nwb.add_acquisition(tps)
+
+    # Processed data
+    if add_processed:
+        print('Adding processed data...')
+        ophys_module = ProcessingModule(
+            name='Ophys',
+            description='contains optical physiology processed data.',
+        )
+        nwb.add_processing_module(ophys_module)
+
+        # Create Image Segmentation compartment
+        img_seg = ImageSegmentation(
+            name=metadata['Ophys']['ImageSegmentation']['name']
+        )
+        ophys_module.add(img_seg)
+
+        # Create plane segmentation and add ROIs
+        meta_ps = metadata['Ophys']['ImageSegmentation']['plane_segmentations'][0]
+        ps = img_seg.create_plane_segmentation(
+            name=meta_ps['name'],
+            description=meta_ps['description'],
+            imaging_plane=nwb.imaging_planes[meta_ps['imaging_plane']],
+        )
+
+        # Add ROIs
+        indices = file_processed_2['indices']
+        indptr = file_processed_2['indptr']
+        dims = np.squeeze(file_processed_1['dims'])
+        for start, stop in zip(indptr, indptr[1:]):
+            voxel_mask = make_voxel_mask(indices[start:stop], dims)
+            ps.add_roi(voxel_mask=voxel_mask)
+
+        # Visualize 3D voxel masks
+        if plot_rois:
+            plot_rois_function(plane_segmentation=ps, indptr=indptr)
+
+        # DFF measures
+        dff = DfOverF(name=metadata['Ophys']['DfOverF']['name'])
+        ophys_module.add(dff)
+
+        # create ROI regions
+        nCells = file_processed_1['dFF'].shape[0]
+        roi_region = ps.create_roi_table_region(
+            description='RoiTableRegion',
+            region=list(range(nCells))
+        )
+
+        # create ROI response series
+        dff_data = file_processed_1['dFF']
+        tt = file_processed_1['time'].ravel()
+        meta_rrs = metadata['Ophys']['DfOverF']['roi_response_series'][0]
+        meta_rrs['data'] = dff_data.T
+        meta_rrs['rois'] = roi_region
+        meta_rrs['timestamps'] = tt
+        dff.create_roi_response_series(**meta_rrs)
+
+        # Creates GrayscaleVolume containers and add a reference image
+        grayscale_volume = GrayscaleVolume(
+            name=metadata['Ophys']['GrayscaleVolume']['name'],
+            data=file_processed_3['im']
+        )
+        ophys_module.add(grayscale_volume)
+
+    # Behavior data
+    if add_behavior:
+        print('Adding behavior data...')
+        # Ball motion
+        behavior_mod = nwb.create_processing_module(
+            name='Behavior',
+            description='holds processed behavior data',
+        )
+        meta_ts = metadata['Behavior']['TimeSeries'][0]
+        meta_ts['data'] = file_processed_1['ball'].ravel()
+        meta_ts['timestamps'] = tt
+        behavior_ts = TimeSeries(**meta_ts)
+        behavior_mod.add(behavior_ts)
+
+        # Re-arranges spatial data of body-points positions tracking
+        pos = file_processed_1['dlc']
+        nPoints = 8
```

Comment:
for variables, can you use underscore_case instead of CamelCase? This should be `npoints` or `n_points`